Mathlib/CategoryTheory/Limits/Shapes/Terminal.lean

```diff
@@ -31,20 +31,33 @@ namespace CategoryTheory.Limits
 
 variable {C : Type u₁} [Category.{v₁} C]
 
-attribute [local tidy] tactic.discrete_cases
+-- attribute [local dee] tactic.discrete_cases -- Porting note: no dee
+open Lean Elab Meta Tactic in 
+-- Porting note: trying a macro 
+/-- A local tactic replacing the use of discrete cases in Lean 3 -/
+elab (name := discrete_empty_elim) "dee" : tactic => withMainContext do
```

Comment:
What does `dee` stand for? Please add the explanation to the docstring.

Reply:
I added this to the docstring.